src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -3998,6 +3998,9 @@ namespace VariableSpeedCoils {
             if ((state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).VSCoilType == DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit) ||
                 (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).VSCoilType == Coil_CoolingAirToAirVariableSpeed)) {
                 for (Mode = 1; Mode <= state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).NumOfSpeeds; ++Mode) {
+                    if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).RatedCapCoolTotal <= 0.0 &&
+                        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).RatedCapCoolTotal != AutoSize)
+                        break;
```

Comment:
At this point SizeVarSpeedCoil has already been called so I don't think && != AutoSize is needed.

Reply:
@rraustad Ok, I think this was probably overly cautious on my part.  I am backing out the != AutoSize part of this.  Seems to be fine with running the tests on EnergyPlus.  I'm running the unit tests now...